doc/introduction/measurements.rst

```diff
@@ -125,6 +124,50 @@ accept observables as arguments,
 including :func:`~.pennylane.expval`, :func:`~.pennylane.var`,
 and :func:`~.pennylane.sample`.
 
+Counts
```

Comment:
Great addition here! :tada: 

It would be good to add `count` to the [summary list of measurements](https://github.com/PennyLaneAI/pennylane/blame/156ccc29f0ac3276e9887a1964d104451947dc5e/doc/introduction/measurements.rst#L32) too.

Reply:
I see it more as an enhancement of a measurement function `sample()` than as a proper measurement, so I added it into the first sentence of the docstring: it will appear in the list, but not as a separate entry. See [this commit](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)

Reply:
Oh, that's a great point! My bad.